Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.m

```diff
@@ -164,7 +164,7 @@ - (CGFloat)bubbleHeight
 
 + (CGFloat)iconSize
 {
-    return 40.f;
+    return 48.f;
```

Comment:
Until I get the new asset from Myles this change will (rightfully) cause an assertion failure, so I won't merge until I get the new asset, but take a look @charlesmchen and let me know if it otherwise looks good.